lib/preprocessors/noop.js

```diff
@@ -0,0 +1,25 @@
+const parsedFiles = new Set();
+
+module.exports = {
+  registerParsedFile(f) {
+    parsedFiles.add(f);
+  },
+  preprocess: null,
+  postprocess: (messages, fileName) => {
+    const msgs = messages.flat();
+    if (
+      msgs.length === 1 &&
+      msgs[0].ruleId === null &&
+      msgs[0].message.startsWith('Parsing error: ')
+    ) {
+      if (!parsedFiles.has(fileName)) {
```

Comment:
this is how we check if the file went through our parser or not.
I only notify the user if a parsing error happened. We could also always notify the user if its not correctly setup.

Reply:
I think this is great! is there a way to check if a gjs / gts file was passed to this though? right now I have a concern about non-gjs and non-gts files hitting this message, would be over-aggressive

Reply:
Well, the preprocessor is only setup for those files